calculator-client/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This XML file contains tabulators. It is not consistent with other source files (for example `calculator-service/pom.xml`) which do not contain tabulators.